src/main.js

```diff
@@ -110,60 +91,72 @@ const startWithMBTiles = (mbtilesFile) => {
   const instance = new MBTiles(mbtilesFile + '?mode=ro', (err) => {
     if (err) {
       console.log('ERROR: Unable to open MBTiles.');
-      console.log(`       Make sure ${path.basename(mbtilesFile)} is valid MBTiles.`);
+      console.log(
+        `       Make sure ${path.basename(mbtilesFile)} is valid MBTiles.`,
+      );
       process.exit(1);
     }
 
     instance.getInfo((err, info) => {
       if (err || !info) {
         console.log('ERROR: Metadata missing in the MBTiles.');
-        console.log(`       Make sure ${path.basename(mbtilesFile)} is valid MBTiles.`);
+        console.log(
+          `       Make sure ${path.basename(mbtilesFile)} is valid MBTiles.`,
+        );
```

Comment:
I still don't like how this breaks up some lines. is this a line length thing?

Reply:
There's space within the ` – https://github.com/maptiler/tileserver-gl/blob/master/src/main.js#L113

Reply:
I've removed this in [58db9b5](https://github.com/maptiler/tileserver-gl/pull/626/commits/58db9b5703f498e2e45d9e8060bcdeb906ae2e4a)